libs/blocks/merch/merch.js

```diff
@@ -122,6 +122,34 @@ const LOADING_ENTITLEMENTS = 'loading-entitlements';
 let log;
 let upgradeOffer = null;
 
+/**
+ * Given a url, calculates the hostname of MAS platform.
+ * Supports, www prod, stage, local and feature branches.
+ * @param {string} hostname
+ * @param {string} maslibs
+ * @returns base url for mas platform
+ */
+export function getMasBase(hostname, maslibs) {
```

Comment:
we will have to move it out of merch since it will be used in other blocks as well (merch-card, etc)
but for now i am fine to keep it here, just a heads up

Reply:
since merch.js will be already loaded regardless, it makes sense to keep it here.
And it is exported, and the value is cached, merch-card.js doesn't need to pass the hostname and maslibs.

Reply:
ahhh true true, nevermind my comment